### PR TITLE
Fixing Redshift config typos

### PR DESCRIPTION
**Pull request by runleonarun**
## What are you changing in this pull request and why?

The code is not versioning or rendering properly:
Screenshot 2023-08-29 at 4 30 35 PM

I think the code snippets need some space.

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `website/static/_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/reference/resource-configs/redshift-configs.md [hunk 1]
```diff
@@ -15,13 +15,18 @@ To-do:
...
+
 <VersionBlock firstVersion="1.6">
+
 - `append` (default)
 - `merge`
 - `delete+insert`
+
 </VersionBlock>
 
 All of these strategies are inheirited via from dbt-postgres.
```
Comment (runleonarun) [inline website/docs/reference/resource-configs/redshift-configs.md]:
```suggestion
All of these strategies are inherited via from dbt-postgres.
```

File: website/docs/reference/resource-configs/redshift-configs.md [hunk 2]
```diff
@@ -15,16 +15,21 @@ To-do:
 In dbt-redshift, the following incremental materialization strategies are supported:
 
 <VersionBlock lastVersion="1.5">
```
Comment (runleonarun) [inline website/docs/reference/resource-configs/redshift-configs.md:17]:
```suggestion
<VersionBlock firstVersion="1.0" lastVersion="1.5">
```

File: website/docs/reference/resource-configs/redshift-configs.md [hunk 3]
```diff
@@ -14,17 +14,22 @@ To-do:
 
 In dbt-redshift, the following incremental materialization strategies are supported:
 
-<VersionBlock lastVersion="1.5">
+<VersionBlock firstVersion="1.0" lastVersion="1.5">
```
Comment (runleonarun) [inline website/docs/reference/resource-configs/redshift-configs.md]:
```suggestion
<VersionBlock lastVersion="1.5">
```

**Review by mirnawong1 — APPROVED** [review]
🚢 and merging this for you @runleonarun

**Pull request merged**